### PR TITLE
Create 2021-11-10-subnero-features-in-sea-technology.md

### PR DESCRIPTION
**Pull request by Florie25**
Hi Anne, could you please review, before I ask Manu and Chinmay to approve it.
Thank you
Florie

### CODE REVIEW

**Comment by AnneThKu** [comment]
The image is not available.

**Comment by Florie25** [comment]
Hi Anne, I added the image now.

**Review by AnneThKu — DISMISSED** [review]

File: _posts/2021-11-10-subnero-features-in-sea-technology.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Subnero featured in Sea Technology magazine
+external_url: https://lsc-pagepro.mydigitalpublication.com/publication/?m=60787&i=727818&p=17&ver=html5/
```
Comment (notthetup) [inline _posts/2021-11-10-subnero-features-in-sea-technology.md]:
```suggestion
external_url: https://lsc-pagepro.mydigitalpublication.com/publication/?m=60787&i=727818&p=17&ver=html5
```

**Comment by notthetup** [comment]
The CI checks are failing. Can we either fix them or remove them?

No point in having failed CI all the time, only makes us insensitive to the CI.

**Comment by notthetup** [comment]
```
> Checking links in _posts/2014-12-03-Singaporean-start-up-Subnero-takes-Internet-underwater.md
> Will allow duplicate links
> Will allow redirects
> Will allow SSL errors
> Connection timeout = 1000s
> White list links matching: http://www.smartseatech.com/en/index.aspx, https://www.asianscientist.com/2017/01/print/hello-world-meet-10-made-in-asia-robots/6/, https://www.asianscientist.com, http://www.oceans19mtsieeemarseille.org/ 
> Will not save results
Links to check: 2, 0 white listed
  1. http://www.koreaherald.com/view.php?ud=20141203000430 
  2. http://www.koreaherald.com/ 
Checking URLs: ??
Issues :-(
> Links 
  1. [L5]  http://www.koreaherald.com/ Failed to open TCP connection to www.koreaherald.com:80 (Connection timed out - connect(2) for "www.koreaherald.com" port 80) 
  2. [L5]  http://www.koreaherald.com/view.php?ud=20141203000430 Failed to open TCP connection to www.koreaherald.com:80 (Connection timed out - connect(2) for "www.koreaherald.com" port 80) 
 ```

**Comment by Florie25** [comment]
Hi @notthetup I often get the CI failure. Do I do something wrong?

**Comment by notthetup** [comment]
The CI is supposed to catch any broken links in all the pulse posts. 

Most of the time flags broken links from old pulse posts where the website we linked to has changed or been taken down by that owner.

We added the CI so we can detect those and fix them. 

Unfortunately the CI can only run when we do an action like raise a PR. So the broken links only get flagged when we want to add/change a page.

I agree it's awkward timing to flag an old broken link. But this is the easiest way to trigger the CI.

**Comment by Florie25** [comment]
Thanks for the explanation @notthetup, I now understand what it means.

**Comment by manuignatius** [comment]
its "featured-in" not "features-in" as the filename.

**Review by manuignatius — DISMISSED** [review]

**Comment by Florie25** [comment]
@manuignatius do you mean "features" or "featured"? The title is "features"

**Comment by manuignatius** [comment]
I was suggesting to use "Subnero featured in..." on both the article header and the file title since it is past tense.

**Review by manuignatius — DISMISSED** [review]

**Review by notthetup — DISMISSED** [review]

**Pull request merged**